web/src/settings_users.js

```diff
@@ -631,3 +638,37 @@ export function set_up_bots() {
         settings_bots.add_a_new_bot();
     });
 }
+
+export function fetch_presence_user_setting(render_table) {
+    if (page_params.is_spectator) {
+        render_table();
+        return;
+    }
+
+    channel.post({
+        url: "/json/users/me/presence",
+        data: {
+            status: compute_active_status(),
+            ping_only: false,
+            last_update_id: -1,
+            history_limit_days: 9999,
+        },
+        success(response) {
+            const data = post_presence_response_schema.parse(response);
+
+            if (data.presences) {
+                user_setting_presence_info.clear();
+
+                for (const [user_id, presence_data] of Object.entries(data.presences)) {
```

Comment:
```suggestion
                for (const [user_id_str, presence_data] of Object.entries(data.presences)) {
                    const user_id = Number.parseInt(user_id_str, 10);
```

you can just do this like in https://github.com/zulip/zulip/blob/94a86d4ac2b68fd7efa3a3bc0b3ee4f0cb18e3f7/web/src/presence.ts#L197-L198 to be consistent with that style and address @sahil839's comment in https://github.com/zulip/zulip/pull/31895#discussion_r1816150382

Reply:
Ohhk, I missed that we parse it to integer in `presence.ts`.